lib/scrabble.rb

```diff
@@ -1,18 +1,47 @@
+require './lib/player'
 class Scrabble
+  attr_reader :players
 
-  def score(word)
-    1
+
+  def initialize(player = Player.new("Bob"))
```

Comment:
Take a look at the interaction pattern. `Scrabble` doesn't initialize with any arguments.